spec.html

```diff
@@ -639,23 +652,8 @@ <h1><ins>ExecuteAsyncModule ( _module_ )</ins></h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getasynccycleroot" aoid="GetAsyncCycleRoot">
-      <h1><ins>GetAsyncCycleRoot ( _module_ )</ins></h1>
-      <emu-alg>
-        1. Assert: _module_.[[Status]] is ~evaluated~.
-        1. If _module_.[[AsyncParentModules]] is an empty List, return _module_.
-        1. Repeat, while _module_.[[DFSIndex]] is greater than _module_.[[DFSAncestorIndex]],
-          1. Assert: _module_.[[AsyncParentModules]] is a non-empty List.
-          1. Let _nextCycleModule_ be the first element of _module_.[[AsyncParentModules]].
-          1. Assert: _nextCycleModule_.[[DFSAncestorIndex]] is less than or equal to _module_.[[DFSAncestorIndex]].
-          1. Set _module_ to _nextCycleModule_.
-        1. Assert: _module_.[[DFSIndex]] is equal to _module_.[[DFSAncestorIndex]].
-        1. Return _module_.
-      </emu-alg>
-    </emu-clause>
-
-    <emu-clause id="sec-asyncmodulexecutionfulfilled" aoid="AsyncModuleExecutionFulfilled">
-      <h1><ins>AsyncModuleExecutionFulfilled ( _module_ )</ins></h1>
+    <emu-clause id="sec-cyclicmoduleexecutionfulfilled" aoid="CyclicModuleExecutionFulfilled">
```

Comment:
Why did the name of this abstract operation change? It doesn't look like you updated usages (e.g., the recursive call).

Reply:
That was a rebase issue, thanks for spotting. I noticed these ids weren't hyphenated per convention so went ahead and added that though.